### PR TITLE
[oracle] go-ora upgrade to 2.7.26 (DBMON-3248)

### PR DESCRIPTION
**Pull request by nenadnoveljic**
### What does this PR do?

Updating `go-ora` drive to the `2.7.26`. Adding `ORA-12514` to the connection errors.

### Motivation

The `go-ora` update was necessary for [adding support for the Korean character set for Windows](https://datadoghq.atlassian.net/browse/SDBM-745). 

### Additional Notes

Previous `go-ora` release contain a  [regression memory leak bug](https://github.com/sijms/go-ora/issues/477), so it was necessary to resolve it, to move forward.

### Describe how to test/QA your changes

Test for memory leak.

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by lu-zhengda — APPROVED** [review]

File: releasenotes/config.yaml
Comment (pgimalac) [inline releasenotes/config.yaml:1]: You probably don't want to delete that :)

Comment (nenadnoveljic) [reply releasenotes/config.yaml:1]: Restored it now, thanks.

File: pkg/collector/corechecks/oracle-dbm/custom_queries.go [hunk 1]
```diff
@@ -166,6 +166,13 @@ func (c *Check) CustomQueries() error {
 							errInQuery = true
 							break
 						}
+					} else if vFloat64, ok := v.(float64); ok {
+						metricRow.value = vFloat64
+						if err != nil {
```
Comment (EmmaOnThursday) [inline pkg/collector/corechecks/oracle-dbm/custom_queries.go:171]: where would this error be coming from? is this accidental copy from L156?

Comment (nenadnoveljic) [reply pkg/collector/corechecks/oracle-dbm/custom_queries.go]: Yes, thanks! Good catch! Also the error handling for `Int64` was unnecessary. Removed now.

**Review by alai97 — APPROVED** [review]
Looks good for docs!

**Review by EmmaOnThursday — APPROVED** [review]

**Pull request merged**